testpilot/frontend/static-src/scripts/legal.js

```diff
@@ -0,0 +1,62 @@
+window.addEventListener('DOMContentLoaded', function (event) {
+  'use strict';
+
+  // Add an accordion to the Privacy Notice page
+  if (location.pathname === '/privacy') {
+    var ps = document.querySelectorAll('h2 ~ p');
+    var tablist = ps[0].parentElement;
```

Comment:
Yep (ref #738). But for now they're in this repo, so we can easily tweak the templates without having to get it reviewed and r+'ed by a different team.